.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,24 @@
+name: Release Drafter
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  update_release_draft:
+    if: github.repository == 'dask/dask'
+    permissions:
+      # Write permission is required to create a GitHub release
+      contents: write
+      pull-requests: read
```

Comment:
Is this needed given we're only running on pushes to `main`?

Reply:
Yes. By default Actions can't read the pull requests API, it's not in the default scope.

Reply:
Hmm TIL, thanks for clarifying